andela/pippin-ah-backend
# Chore/162727544/configure houndci

#### What does this PR do?
This PR integrates houndci with the repo

#### Description of Task to be completed?
- Add .houndci.yml file to the root directory
- Add configuration code to .houndci.yml file
- Add houndci badge to readme.md

#### How should this be manually tested?
- View readme.md to see houndci badge
- Open a GitHub pull request
- Hound will review the pull request and comment inline on any style violation as declared in .eslint file

#### What are the relevant pivotal tracker stories?
The relevant pivotal tracker story for this PR:
 - #162727544 https://www.pivotaltracker.com/n/projects/2229853/stories/162727544
